sdk/core/System.Net.ClientModel/src/ModelReaderWriter/ModelReaderWriterOptions.cs

```diff
@@ -21,6 +21,11 @@ public class ModelReaderWriterOptions
         /// </summary>
         public static readonly ModelReaderWriterOptions DefaultWireOptions = _singletonMap[ModelReaderWriterFormat.Wire];
 
+        /// <summary>
+        /// Default options for writing models into the JSON format.
+        /// </summary>
+        public static readonly ModelReaderWriterOptions DefaultJsonOptions = _singletonMap[ModelReaderWriterFormat.Json];
```

Comment:
Do we need this?  For the factory methods, shouldn't we we using `DefaultWireOptions` since they are "what the service expects"?  @m-nash, I defer to you on this - I don't have a great understanding still of the difference between `Wire` and `Json`.

Reply:
No, for the factory methods we should use Json format. It corresponds to the API shape and so users can easily create anonymous instance when mocking. If I used the wire format, the anonymous type created in the test (see the unit test) would have to mirror the wire format which nobody knows what it is.

Reply:
That makes sense.

One question I had, although this may not be the right place to ask, is ... how are we validating that all models will implement all relevant formats correctly?  We may be able to encode it in the generator, but what if a community user wants to implement it in a hand-written model in a hand-written client?  What if we add a new value to Options, will people be expected to update their model types that implement the interfaces to address it?

The reason I ask here is because .. how can we be sure that the `Json` format will always be supported in an implementation of `IJsonModel<T>`?